R/api-bump-version.R

```diff
@@ -5,6 +5,28 @@
 #'
 #' @inheritParams update_version
 #' @export
+#'
+#' @examples
+#' # Create temporary directory to hold a mock package.
+#' tempdir <- file.path(tempdir(check = TRUE), "fledge")
+#' dir.create(tempdir)
+#' # Create said mock package.
+#' # Set open to TRUE if you want to play in the mock package.
+#' pkg <- create_demo_project(open = FALSE, news = TRUE, tempdir = tempdir)
+#' # Use functions as if inside the newly created package project.
+#' # (Or go and actually run code inside the newly created package project!)
+#' usethis::with_project(pkg, {
+#' # Add a new R file.
+#' usethis::use_r("cool-function", open = FALSE)
+#' # Pretend we added useful code inside it.
+#' # Track the new R file with Git.
+#' gert::git_add("R/cool-function.R")
+#' gert::git_commit("- Add cool function.")
+#' # Bump version with fledge.
+#' fledge::bump_version()
```

Comment:
Namespacing for the case the user is running this line by line in the new mock project (where fledge isn't necessarily loaded yet).